build.sh

```diff
@@ -1,85 +1,105 @@
-#!/bin/bash
+#!/bin/sh
 
 # Stop on first error
 set -e
 
 #SEQ=!!!!! ENTER YOUR FAVORITE Y4M HERE !!!!!
 
-IS_RELEASE=1
+unset IS_RELEASE
 
 for arg in "$@"; do
-  shift
-  case "$arg" in
-    "--debug") IS_RELEASE=0 ;;
-    *)        set -- "$@" "$arg"
+  case $arg in
+  --debug) unset IS_RELEASE ;; # Uneeded, but explicit
+  --release) IS_RELEASE=true ;;
+  *) continue ;;
   esac
 done
 
-if [[ -z "${SEQ}" ]]; then
+if [ -z "$SEQ" ]; then
   SEQ=nyan.y4m
-  SEQ10=nyan10.y4m
-  SEQ12=nyan12.y4m
-
-  wget -nc https://mf4.xiph.org/~ltrudeau/videos/nyan.y4m
+  #SEQ10=nyan10.y4m
+  #SEQ12=nyan12.y4m
+
+  if type wget > /dev/null 2>&1; then
+    wget -nc https://mf4.xiph.org/~ltrudeau/videos/nyan.y4m
+  elif type curl > /dev/null 2>&1; then
+    curl -fkLs https://mf4.xiph.org/~ltrudeau/videos/nyan.y4m -o nyan.y4m
+  fi
```

Comment:
We could error here or at least warn the user that we expected wget or curl.

Reply:
Sure